src/Config/ValueProvider/SourceDirsProvider.php

```diff
@@ -60,6 +58,11 @@ public function get(InputInterface $input, OutputInterface $output, array $dirsI
 
         $choices = array_merge(['.'], array_values($dirsInCurrentDir));
 
+        // if a default value is not any of current choices, add its components to allowed values
+        if (!in_array($defaultValues, $choices)) {
```

Comment:
shouldn't it be

```
foreach (guessedSourceDirs as $guessedSourceDir) {
    if (!in_array($defaultValues, $choices, true)) {
        $choices[] = $guessedSourceDir;
    }
}
```

?

What I mean here is we need to check all guessed values separately, not as concatenated string `src/A,src/B`